spot-client/src/spot-tv/ui/components/meeting-frame/KickedOverlay.js

```diff
@@ -11,15 +12,18 @@ import { Countdown, StatusOverlay } from 'common/ui';
  * instance is to be initialized.
  * @returns {ReactNode}
  */
-export function KickedOverlay(props) {
+export function KickedOverlay({ onRedirect, t }) {
     return (
-        <StatusOverlay title = 'You have been removed from the conference'>
-            <div>The conference will be exited automatically in:</div>
-            <Countdown onCountdownComplete = { props.onRedirect } />
+        <StatusOverlay title = { t('conferenceStatus.kicked') }>
+            <div>{ t('appStatus.redirectingHome') }</div>
```

Comment:
Somehow the old 'The conference will be exited automatically in:' sounds better than 'Redirecting to home in` to me?